sdks/java/io/hadoop-input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/unit/tests/coders/EmployeeCoder.java

```diff
@@ -0,0 +1,64 @@
+package org.apache.beam.sdk.io.hadoop.inputformat.unit.tests.coders;
+
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.beam.sdk.coders.Coder;
+import org.apache.beam.sdk.coders.CoderException;
+import org.apache.beam.sdk.coders.StandardCoder;
+import org.apache.beam.sdk.coders.StringUtf8Coder;
+import org.apache.beam.sdk.io.hadoop.inputformat.unit.tests.HadoopInputFormatIOTest;
+import org.apache.beam.sdk.io.hadoop.inputformat.unit.tests.HadoopInputFormatIOTest.Employee;
+
+
+public class EmployeeCoder extends StandardCoder<Employee> implements Serializable{
```

Comment:
we should be able to use avrocoder/serializablecoder and not need to implement a new coder for this type.